05-Mining-Protocol.md

```diff
@@ -485,11 +485,11 @@ A proxy MUST translate the message for all downstream channels belonging to the
 - For an **extended channel**: `extranonce_prefix + extranonce (=N bytes)`, where `N` is the negotiated extranonce space for the channel (`OpenMiningChannel.Success.extranonce_size`)
 
 ### 5.3.17 `SetNewPrevHash` (Server -> Client, broadcast)
-Prevhash is distributed whenever a new block is detected in the network by an upstream node.
+Prevhash is distributed whenever a new block is detected in the network by an upstream node or when a new downstream opens a channel.
 This message MAY be shared by all downstream nodes (sent only once to each channel group).
 Clients MUST immediately start to mine on the provided prevhash.
 When a client receives this message, only the job referenced by Job ID is valid.
-The remaining jobs already queued by the client have to be made invalid. 
+The remaining jobs already queued by the client have to be made invalid unless they are future jobs. 
```

Comment:
This addition is not true. When a SetNewPrevHash all the future jobs must be removed and are no more valid.
Note that the only job that are queued are future jobs.  Non future jobs do not get queued but are executed immediately

Reply:
@darricksee Can you correct this?

Reply:
Done